crates/controller/src/main.rs

```diff
@@ -193,8 +189,8 @@ async fn reconcile(
                         SessionLivedBackendStatus::default(),
                     )
                     .await?;
-                },
-                _ => ()
+                }
```

Comment:
Oh interesting - how does this behave without the `,`?

Reply:
The `,` in a `match` statement can be omitted after a `}` or after the last element. It doesn't change the meaning. In this case, `cargo fmt` decided to remove it.